src/porepy/numerics/fv/biot.py

```diff
@@ -983,4 +1051,17 @@ def assemble_rhs(self, g, data):
             np.ndarray: Zero right hand side vector with representation of boundary
                 conditions.
         """
-        return np.zeros(self.ndof(g))
+        parameter_dictionary = data[pp.PARAMETERS][self.keyword]
+        matrix_dictionary = data[pp.DISCRETIZATION_MATRICES][self.keyword]
+
+        # The stabilization is the pressure contribution to the div u part of the
+        # fluid mass conservation, thus need a right hand side in the implicit Euler
+        # discretization.
+        pressure_0 = parameter_dictionary["state"]
```

Comment:
This is the pressure at the previous time step? If so, I think it should not be in the parameter dictionary. @IvarStefansson @rbe051 What do you say?

Reply:
We should anyhow not have a parameter or variable called state.